lib/src/modules/task.cc

```diff
@@ -583,6 +680,14 @@ ActionResponse Task::callAction(const ActionRequest& request)
                                        client_,
                                        file);
 
+    auto meta_files = task_metadata.getWithDefault<std::vector<lth_jc::JsonContainer>>("files", {});
+    auto lib_files = getMultiFiles(meta_files, implementation.files, files);
+    if (lib_files.size() > 0) {
+        install_dir = downloadMultiFile(files, download_set);
+    }
+
+    // TODO pass install_dir as metaparameter _installdir for task execution
```

Comment:
Will need to pass this location as _installd_dir to the task. Will also need to delete the install_dir after execution (including if the task fails).

Reply:
We don't need to delete the install dir after execution. spooldir's normal cleanup will handle that.